flink-runtime/src/test/java/org/apache/flink/runtime/dispatcher/TestingJobManagerRunnerFactory.java

```diff
@@ -82,10 +83,10 @@ public JobManagerRunner createJobManagerRunner(
 		} else {
 			jobGraphFuture.complete(jobGraph);
 
-			final JobManagerRunner mock = mock(JobManagerRunner.class);
```

Comment:
It can remain to be `JobManagerRunner` here.
But I think it's fine to since PR#9789 will refactor it later.

Reply:
Yes exactly. Will keep it since #9789 should be merged soon.